crates/python/qcs_sdk/tracing_subscriber/__init__.pyi

```diff
@@ -0,0 +1,119 @@
+# *****************************************************************************
+# *                             AUTO-GENERATED CODE                           *
+# *                                                                           *
+# * This code was generated by the `pyo3-tracing-subscriber` crate. Any       *
+# * modifications to this file should be made to the script or the generation *
+# * process that produced this code. Specifically, see:                       *
+# * `pyo3_tracing_subscriber::stubs::write_stub_files`                        *
+# *                                                                           *
+# * Do not manually edit this file, as your changes may be overwritten the    *
+# * next time the code is generated.                                          *
+# *****************************************************************************
+
+from types import TracebackType
+from typing import Optional, Type, Union
+from . import subscriber as subscriber
+from . import layers as layers
+
+class TracingContextManagerError(RuntimeError):
+    """
+    Raised if the initialization, enter, and exit of the tracing context manager was
+    invoked in an invalid order.
+    """
+
+    ...
+
+class TracingStartError(RuntimeError):
+    """
+    Raised if the tracing subscriber configuration is invalid or if a background export task
+    fails to start.
+    """
+
+    ...
+
+class TracingShutdownError(RuntimeError):
+    """
+    Raised if the tracing subscriber fails to shutdown cleanly.
+    """
+
+    ...
+
+class BatchConfig:
```

Comment:
These type hints do not pass [stubtest](https://mypy.readthedocs.io/en/stable/stubtest.html). There are 65 in total, but for an example, this one fails with:

```
error: qcs_sdk.BatchConfig is not present in stub
Stub: in file .../qcs-sdk-rust/crates/python/qcs_sdk/__init__.pyi
```

When you are in `crates/python`: `cargo make stubtest-flow` will install the module and run stubtest and `cargo make stubtest` will just run `stubtest`. The latter is helpful when you are only iterating in the `pyi` files.

Reply:
Ahh, did not know about this utility. I will fix it upstream and port it over here.

Reply:
Finally got those passing after a good tussle. They actually did help me identify a bug though.